src/components/exchange/CurrencySelectionList.js

```diff
@@ -45,7 +44,7 @@ const CurrencySelectionList = (
   ref
 ) => {
   const skeletonTransitionRef = useRef();
-  const noResults = get(listItems, '[0].data', []).length === 0;
+  const noResults = (listItems?.[0]?.data ?? []).length === 0;
```

Comment:
```suggestion
  const noResults = listItems?.[0]?.data?.length ?? 0 === 0;
```

or

```suggestion
  const noResults = !!listItems?.[0]?.data?.length
```

Reply:
Good point, thanks. Will use with one NOT for the same result
`const noResults = !listItems?.[0]?.data?.length`